services/pom.xml

```diff
@@ -185,6 +185,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Those dependency may not be needed. I tested and it was ok. In the dependency tree, the slf4j-api is already on the project's scope:
exo@MEA-054-L:~/eXoPlatform/eXoProjects/gamification/services (git:US5_Implement_UT)$ mvn dependency:tree | grep 'slf4j'
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=1G; support was removed in 8.0
[INFO] |  \- org.slf4j:slf4j-api:jar:1.7.18:compile